cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationController.java

```diff
@@ -0,0 +1,50 @@
+package mil.army.usace.hec.cwms.radar.client.controllers;
```

Comment:
this file isnt related to your change

Reply:
Once PR6 is merged, i can do another rebase off main, which will remove this